Sources/PointFree/Account/Change.swift

```diff
@@ -322,7 +322,7 @@ private let individualPricingColumnView = View<(Pricing.Billing, Pricing)> { bil
             ]),
           ]),
         gridColumn(sizes: [.mobile: 12], [], [
-          h6([`class`([Class.pf.type.title6, Class.pf.colors.fg.gray650, Class.display.inline])], [
+          h6([`class`([Class.pf.type.responsiveTitle7, Class.pf.colors.fg.gray650, Class.display.inline])], [
```

Comment:
screenshots saved my butt here! note that `title6` in the old system was the small ALL CAPS style, but in the new responsible system it's `responsiveTitle7`. I didn't realize that until I saw the different in the screenshots.